doc/source/guides/common-tasks.rst

```diff
@@ -59,9 +62,10 @@ document from an image.
 
     ...
 
+Upload a document without meta information
+==========================================
 
-
-Read on to find out how to get the extractions from a document.
+To create a document from an image without sending meta infromation use the ``createDocumentWithFilename:fromImage:docType:`` method instead. This method takes a filename, an instance of `UIImage` and a hint to the type of document which is to be analyzed as parameters. 
```

Comment:
This method takes as parameters a filename, an instance of `UIImage` and a hint for the type of document to be analyzed.